### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by aashikvilla**

### CODE REVIEW

File: src/NotesApp.Api/Program.cs [hunk 1]
```diff
@@ -1,14 +1,57 @@
...
 
 // Add services to the container.
 
 builder.Services.AddControllers();
+builder.Services.AddDbContext<AppDbContext>(options => options.UseSqlServer(connectionString));
+
+builder.Services.AddScoped<IUserService, UserService>();
+builder.Services.AddScoped<IUserRepository, UserRepository>();
+builder.Services.AddScoped<IPasswordHasher, PasswordHasher>();
+builder.Services.AddScoped<ITokenService, TokenService>();
```
Comment (rohitvipin) [inline src/NotesApp.Api/Program.cs]: Why is everything scoped?

File: src/NotesApp.Application/Common/ResponseMessages.cs [hunk 2]
```diff
@@ -0,0 +1,12 @@
+﻿namespace NotesApp.Application.Common
+{
+    public static class ResponseMessages
```
Comment (rohitvipin) [inline src/NotesApp.Application/Common/ResponseMessages.cs]: Prefer resx file

File: src/NotesApp.Application/Response/ServiceResponse.cs [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NotesApp.Application.Response
+{
+    public class ServiceResponse<T>
+    {
+        public bool Success { get; set; } = false;
```
Comment (rohitvipin) [inline src/NotesApp.Application/Response/ServiceResponse.cs]: Not a fan of this. 
for errors throw exception 
Only success should return expected type.

File: NotesApp.sln [hunk 4]
```diff
@@ -11,7 +11,9 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "NotesApp.Infrastructure", "
 EndProject
```
Comment (kool7) [inline NotesApp.sln:11]: test comment

File: src/NotesApp.Api/Program.cs (hunk 1 above)
Comment (aashikvilla) [reply src/NotesApp.Api/Program.cs]: changed it to transient

File: src/NotesApp.Application/Response/ServiceResponse.cs (hunk 3 above)
Comment (aashikvilla) [reply src/NotesApp.Application/Response/ServiceResponse.cs]: removed service response and updated the code with exceptions

File: src/NotesApp.Application/Common/ResponseMessages.cs (hunk 2 above)
Comment (aashikvilla) [reply src/NotesApp.Application/Common/ResponseMessages.cs]: Added resx file for constants

**Pull request closed without merging**